Mathlib/LinearAlgebra/TensorPower.lean

```diff
@@ -299,4 +299,21 @@ theorem galgebra_toFun_def (r : R) :
 
 example : Algebra R (⨁ n : ℕ, ⨂[R]^n M) := by infer_instance
 
+variable (n : ℕ)
+
+variable (R)
+
+/-- A family `f` of linear forms on `M` indexed by `Fin n` defines a linear form on
+`⨂[R]^n M`, by multiplying the components of `f`. -/
+noncomputable def linearFormOfFamily (f : (_ : Fin n) → (M →ₗ[R] R)) : (⨂[R]^n) M →ₗ[R] R :=
```

Comment:
```suggestion
noncomputable def linearFormOfFamily (f : (_ : Fin n) → Module.Dual R M) : Module.Dual R (⨂[R]^n M) :=
```

I think this holds more generally for the n-ary tensor products, rather than just tensor powers; can you add that definition instead?

Reply:
I don't think that it should make it more complicated, so I'll try.
You suggested having a name that contains `dual` instead of `linearFormOfFamily`, and I'm not against that, but I don't feel very inspired.

Reply:
[`TensorProduct.dualDistrib`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/LinearAlgebra/Dual.html#TensorProduct.dualDistrib) seems pretty close; ideally we would bundle this map slightly further, and use the same name.

Reply:
You're right, my `linearFormOfFamily` is basically `PiTensorProduct.dualDistrib` applied to the case of a constant family. `PiTensorProduct.dualDistrib` doesn't exist yet, but I'll get to work on it. 
I don't know if you were also suggesting to unify the case of the tensor product of two modules and that of the tensor product of a finite family of modules; if so, I don't know how to do that.

Reply:
I started working in [a new branch](https://github.com/leanprovercommunity/mathlib4/tree/SM.PiTensorProduct). Let's hope this doesn't grow too large.

Reply:
> I don't know if you were also suggesting to unify the case of the tensor product of two modules and that of the tensor product of a finite family of modules; if so, I don't know how to do that.


No, I am certainly not suggesting this! Just that we should try to match the names of the AP

Reply:
> I started working in [a new branch](https://github.com/leanprovercommunity/mathlib4/tree/SM.PiTensorProduct). Let's hope this doesn't grow too large.

It looks like you haven't pushed this branch yet.

Reply:
I did push it, but I didn't create a PR yet.